grid/vcs/repository-files/maps/github.test.ts

```diff
@@ -0,0 +1,89 @@
+import { SuperfaceTest } from '@superfaceai/testing';
+
+describe(`vcs/repository-files/github`, () => {
+  let superface: SuperfaceTest;
+
+  beforeEach(() => {
+    superface = new SuperfaceTest({
+      profile: 'vcs/repository-files',
+      provider: 'github',
+      testInstance: expect,
+    });
+  });
+
+  describe('ListDirectory', () => {
+    it('lists files without path and reference', async () => {
+      await expect(
+        superface.run({
+          useCase: 'ListDirectory',
+          input: {
+            owner: 'octocat',
+            repository: 'hello-world',
+          },
+        })
+      ).resolves.toMatchSnapshot();
+    });
+    it('lists a single file inside entries', async () => {
+      await expect(
+        superface.run({
+          useCase: 'ListDirectory',
+          input: {
+            owner: 'octocat',
+            repository: 'hello-world',
+            path: 'README',
+          },
+        })
+      ).resolves.toMatchSnapshot();
+    });
+
+    it('accepts a repository identifier without owner', async () => {
+      const result = await superface.run({
+        useCase: 'ListDirectory',
+        input: {
+          repository: 'octocat/hello-world',
+        },
+      });
+
+      expect(result.isOk()).toBe(true);
+    });
+
+    it('accepts a git reference', async () => {
+      const result = await superface.run({
+        useCase: 'ListDirectory',
+        input: {
+          repository: 'octocat/Spoon-Knife',
+          reference: 'test-branch',
+        },
+      });
+      expect(result.isOk()).toBe(true);
+      expect(result).toMatchSnapshot();
+    });
+
+    describe('errors handling', () => {
```

Comment:
Maybe add test case for undefined owner? 🤔

Reply:
Added!